packages/file_selector/file_selector/example/lib/get_directory_page.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:file_selector/file_selector.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that shows an example of openFile(s)
```

Comment:
This screen shows an example of getDirectoryPath, it seems :P

Reply:
Great catch! Fixed here and (hopefully) everywhere.